src/dashboard.Dockerfile

```diff
@@ -19,13 +19,64 @@ RUN set -ex \
 		locales \
 		locales-all \
 		unar \
+		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
 # Set the locale
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
+#
+# Chrome for Karma JS tests
+#
+
+ARG CHROME_VERSION="google-chrome-beta"
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+	&& echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
+	&& apt-get update -qqy \
+	&& apt-get -qqy install ${CHROME_VERSION:-google-chrome-stable} \
+	&& rm /etc/apt/sources.list.d/google-chrome.list \
+	&& rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+ARG CHROME_DRIVER_VERSION="latest"
+RUN CD_VERSION=$(if [ ${CHROME_DRIVER_VERSION:-latest} = "latest" ]; then echo $(wget -qO- https://chromedriver.storage.googleapis.com/LATEST_RELEASE); else echo $CHROME_DRIVER_VERSION; fi) \
+	&& echo "Using chromedriver version: "$CD_VERSION \
+	&& wget --no-verbose -O /tmp/chromedriver_linux64.zip https://chromedriver.storage.googleapis.com/$CD_VERSION/chromedriver_linux64.zip \
+	&& rm -rf /opt/selenium/chromedriver \
+	&& unzip /tmp/chromedriver_linux64.zip -d /opt/selenium \
```

Comment:
I'm seeing this error:

    /bin/sh: 1: unzip: not found

It disappear when I had `unzip` installed. It'd also be possible to rewrite the snippet so it uses `unar` instead, which is already installed by us (see L21).